src/audio/asrc/asrc.c

```diff
@@ -419,7 +419,7 @@ static int asrc_params(struct comp_dev *dev,
 	 */
 	cd->source_frames_max = cd->source_frames + 10;
```

Comment:
What does it mean 'Zephyr compatibility' ? Is Zephyr using Z_MAX naming instead of MAX?